### PR TITLE
Add script to run in Docker

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: bin/run-in-docker.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Build the Docker image
+docker build --rm -t exercism/test-runner .
+
+# Run the Docker image using the settings mimicking the production environment
+docker run \
+    --network none \
+    --read-only \
+    --mount type=bind,src="${input_dir}",dst=/solution \
+    --mount type=bind,src="${output_dir}",dst=/output \
+    --mount type=tmpfs,dst=/tmp \
```
Comment (ekingery) [inline bin/run-in-docker.sh:41]:
```suggestion
    --mount type=bind,src=/tmp,dst=/tmp \
```

This seems to resolve the issue with the empty output. I'm _guessing_ due to the `ENV GOCACHE=/tmp` line in the Dockerfile. Not sure if this is the solution we want, but maybe it's good enough for now?

**Review by ekingery — CHANGES_REQUESTED** [review]
LGTM, pending the issue with the tmp mount

File: bin/run-in-docker.sh (hunk 1 above)
Comment (ErikSchierboom) [reply bin/run-in-docker.sh]: On second thought, that doesn't work on systems that don't have a /tmp directory. Let me look into it.

Comment (ErikSchierboom) [reply bin/run-in-docker.sh]: I've found a different way to fix it, using a volume mount

**Review by ekingery — APPROVED** [review]
Works for me on my local, and I like mounting /tmp as a volume instead of binding it. Thanks Erik!

**Comment by ErikSchierboom** [comment]
@iHiD Could you approve?

**Review by iHiD — APPROVED** [review]

**Pull request merged**